### PR TITLE
Workaround for login/signup styles different in production

### PR DESCRIPTION
**Pull request by lucaslcode**
Unfortunately this requires manually adding `{index: 1}` to affected styles, so I guess we'll just have to keep an eye out...

### CODE REVIEW

**Comment by aapeliv** [comment]
Glad you found a workaround!


Just an idea looking at this: are you able to patch this everywhere by changing imports and wirting a shim around the original `makeStyles` like this:

```ts
makeStyles = (originalMakeStyles: (theme) => styleType) => (theme => ({originalMakeStyles(theme), {index: 1}))
```

This probably doesn't run, but maybe you get what I mean? Might be less effort and fix it everywhere?

**Comment by lucaslcode** [comment]
Thank god for regex

**Comment by darrenvong** [comment]
Does this affect the ordering in which the the styles are injected only? Also where is this documented? Did you have to dig into their source code for this?

File: app/frontend/src/components/Divider.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { Divider as MuiDivider, makeStyles } from "@material-ui/core";
+import { Divider as MuiDivider,  } from "@material-ui/core";
```
Comment (darrenvong) [inline app/frontend/src/components/Divider.tsx]:
```suggestion
import { Divider as MuiDivider } from "@material-ui/core";
```

**Review by darrenvong — COMMENTED** [review]
The formatting on a few files looks a bit off (I didn't bother commenting on all of them) - might want to run Prettier over everything?

**Comment by lucaslcode** [comment]
https://cssinjs.org/jss-api?v=v10.6.0#create-style-sheet
Higher index means lower in `<head>`, and it defaults to 0

**Review by darrenvong — APPROVED** [review]
Seems like a perfectly fine solution to me than a workaround especially as we made it into a helper function! 🎉

**Pull request merged**